### PR TITLE
fix(weaver-satp): sample SATP implementation fixes and miscellaneous cleanups

### PR DESCRIPTION
**Pull request by VRamakrishna**
Modified the SATP trigger protobuf message.
Fixed and augmented the satpsimpleasset chaincode to support SATP operation.
Fixed and augmented the Fabric interop SDK to support SATP operation.
Deleted obsolete files in the testnet folders and fixed minor typos in docs.
Reissued expired sample certificates and keys and bug fixes.
Sample certificates and keys used in dockerized relays, drivers, and iin-agents reissued with 5-year validity.
Fixed a configuration bug in the IIN Agent configs whereby a string value was writted mistakenly as a boolean.
Removed unnecessary deprecated 'start()' call for gRPC servers in Node services (Fabric driver, IIN-agent, SDK unit test).

**Pull Request Requirements**
- [ ] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [ ] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [ ] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [ ] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [ ] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

File: weaver/core/drivers/fabric-driver/server/sample-satp.ts [hunk 1]
```diff
@@ -215,102 +212,62 @@ async function assignAssetHelper(
...
-    }
-    hash.setPreimage(secret)
-
-    let contractId: string = null
-    if (assignAssetRequest2['contract-id']) {
-        contractId = assignAssetRequest2['contract-id']
-    }
-
     const params = assignAssetRequest2['param'].split(':')
+    const walletPath = process.env.WALLET_PATH
```
Comment (sandeepnRES) [inline weaver/core/drivers/fabric-driver/server/sample-satp.ts:239]: Can you add a TODO here too, that to read recipient cert from request parameters.

**Review by sandeepnRES — APPROVED** [review]
LGTM

**Review by petermetz — APPROVED** [review]
@VRamakrishna LGTM but please try to squash the commits together.

File: weaver/core/drivers/fabric-driver/server/sample-satp.ts (hunk 1 above)
Comment (VRamakrishna) [reply weaver/core/drivers/fabric-driver/server/sample-satp.ts:241]: Done

**Pull request merged**